src/controllers/symptoms.js

```diff
@@ -27,16 +25,17 @@ exports.get = (req, res) => {
 exports.post = (req, res) => {
   saveSection(req.session.id, 'symptoms', req.body)
     .then(() => {
-      res.redirect('about');
+      const buttonPressed = req.body.button;
+      if (buttonPressed === 'next') {
+        res.redirect('about');
+      } else if (buttonPressed === 'previous') {
+        res.redirect('appointments');
+      } else {
+        res.redirect('symptoms');
+      }
```

Comment:
Something to note, the code here is identical to background.js might be worth thinking about how you would make it DRY. Happy to pull this in for now. :)